test/integration_tests/prototype/ctc_decoder_integration_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from torchaudio.prototype.ctc_decoder import lexicon_decoder, download_pretrained_files
+
+
+@pytest.mark.parametrize("model", ["librispeech", "librispeech-3-gram"])
+def test_download_pretrain(model):
+    # smoke test for downloading pretrained files
+    _ = download_pretrained_files(model)
+
+
+@pytest.mark.parametrize("model", ["librispeech", "librispeech-3-gram"])
+def test_decoder_from_pretrained(model):
```

Comment:
I think `test_download_pretrain` and `test_decoder_from_pretrained` can be merged. `test_download_pretrain` does not seem to add much value over `test_decoder_from_pretrained `, and yet it is hard to decouple the download logic from `test_decoder_from_pretrained`.

Reply:
Can we put `from torchaudio.prototype.ctc_decoder import lexicon_decoder, download_pretrained_files` inside of the test function so that the test will fail at test time instead of collect time, in case the said module is not available.

Reply:
good point, thanks